### DISCUSSION TITLE
放只万穗爷在这

### DISCUSSION THREAD

**Discussion by lyk758811** [share121/inter-knot#1665] [其他]
![6DC501B9F79BDC9F48DAEB1D44823F20](https://github.com/user-attachments/assets/5de424e5-7a1d-4d41-bf75-db8a8c16c0cd)
![3A8BC2F765C62A07B40659D319E53CA2](https://github.com/user-attachments/assets/c48223c3-ab9a-4cb0-8844-e33f8bb6717f)

**Comment by xioagong** [comment]
见识过星星粉碎的样子吗（）

**Comment by Elysia-Eden-11** [comment]
你是什么东西？也以世界之名，世界之名是我布洛........

**Comment by Lingyao-cangxuan** [comment]
明末耐掐王
